### PR TITLE
Emcs 330

### PR DESCRIPTION
**Pull request by mauro-fanton**

### CODE REVIEW

File: it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala [hunk 1]
```diff
@@ -366,18 +366,21 @@ class PollingNewMessagesWithWorkItemJobItSpec extends PlaySpec
...
-      workItem.availableAt mustBe createdWorkItem.availableAt.plusSeconds(2 * 60).truncatedTo(ChronoUnit.MILLIS)
+        eventually {
+          verifyItemStatus(ProcessingStatus.ToDo).futureValue mustBe true
+        }
 
       }
     }
 
-    "if fails mark work item as failed so can be retried" in {
+    //todo: This test seems to be the same as the above one. Can it be removed?
```
Comment (ChrisCookOC) [inline it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala:382]: I think it was originally meant to from the title saying it should be failed but the implementation is the same. So need to either make this test actually test it failed - not sure how as the polling will retry it - or remove it.

File: it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala.orig [hunk 2]
```diff
@@ -0,0 +1,730 @@
...
+import uk.gov.hmrc.mongo.workitem.{ProcessingStatus, WorkItem}
+
+import java.nio.charset.StandardCharsets
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.Base64
+import scala.concurrent.duration.{DAYS, Duration, MINUTES}
+import scala.concurrent.{ExecutionContext, Future}
+
+class PollingNewMessagesWithWorkItemJobItSpec extends PlaySpec
```
Comment (ChrisCookOC) [inline it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala.orig:54]: you have this file twice?

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -2,28 +2,31 @@ import sbt._
...
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-28"    % mongoVersion,
+  val test: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"             %% "bootstrap-test-play-30"     % bootstrapVersion,
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-30"    % mongoVersion,
     "org.scalatestplus"       %% "scalatestplus-mockito"      % "1.0.0-M2",
     "org.mockito"             %% "mockito-scala"              % "1.17.12"
-  ).map(_ % "test, it")
+  ).map(_ % "test")
+
+  val overrides: Seq[ModuleID] = Seq(
```
Comment (ChrisCookOC) [inline project/AppDependencies.scala:29]: What's this for, out of interest

**Review by ChrisCookOC — CHANGES_REQUESTED** [review]

File: project/AppDependencies.scala (hunk 3 above)
Comment (mauro-fanton) [reply project/AppDependencies.scala:29]: I think the new play-30 depends from scala.2.13 V2.2.0, but play bring in a lower version of scala-xml, so we need to override this to get the right version

File: it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala (hunk 1 above)
Comment (mauro-fanton) [reply it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala:382]: I will leave a todo to look at this test at later time.

File: it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala.orig (hunk 2 above)
Comment (mauro-fanton) [reply it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala.orig]: yes, this is ot supposed to be there. This is a leftover of git rebase that I have added to the repo by mistake. Removed.

File: it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala (hunk 1 above)
Comment (mauro-fanton) [reply it/test/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PollingNewMessagesWithWorkItemJobItSpec.scala]: removed the duplicate test now

**Review by ChrisCookOC — APPROVED** [review]

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**